### PR TITLE
Added favouriting of events

### PR DESCRIPTION
**Pull request by Requinard**
title says all

### CODE REVIEW

File: app/src/main/kotlin/org/eurofurence/connavigator/database/UpdateIntentService.kt [hunk 1]
```diff
@@ -100,6 +100,6 @@ class UpdateIntentService(val api: DefaultApi = DefaultApi()) : IntentService("U
         }
 
         // Send a broadcast notifying completion of this action
-        LocalBroadcas   tManager.getInstance(this).sendBroadcast(response)
```
Comment (lukashaertel) [inline app/src/main/kotlin/org/eurofurence/connavigator/database/UpdateIntentService.kt]: what happened!

File: app/src/main/kotlin/org/eurofurence/connavigator/util/Favoriter.kt [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        val event_date = database.eventConferenceDayDb[eventEntry.conferenceDayId]
+        val event_start_time = eventEntry.startTime
+
+        //format will be "yyyy-MM-dd hh:mm:ss"
+        val eventDateTimeString = "%s %s".format(event_date!!.date, event_start_time)
+        val eventDateTimeFormatter = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss")
+
+        val eventDateTime = eventDateTimeFormatter.parseDateTime(eventDateTimeString)
+
+        eventDateTime.minusMinutes(30)
```
Comment (lukashaertel) [inline app/src/main/kotlin/org/eurofurence/connavigator/util/Favoriter.kt]: Hardcoding constants, classy ... :+1:

Comment (Requinard) [inline app/src/main/kotlin/org/eurofurence/connavigator/util/Favoriter.kt]: I'd like to thank my mom and my dad

File: app/src/main/kotlin/org/eurofurence/connavigator/database/UpdateIntentService.kt (hunk 1 above)
Comment (Requinard) [inline app/src/main/kotlin/org/eurofurence/connavigator/database/UpdateIntentService.kt]: idunno but it fixed

**Comment by lukashaertel** [comment]
Go merge, is ok >3>

**Pull request merged**